pkgs/by-name/ko/komac/package.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, darwin
+, testers
+, komac
+}:
+
+let
+  version = "2.1.0";
+  src = fetchFromGitHub {
+    owner = "russellbanks";
+    repo = "Komac";
+    rev = "v${version}";
+    hash = "sha256-L8UYpNqjRyqf4hPQwD9LaXWu6jYaP34yTwTxcqg+e2U=";
+  };
+in
+rustPlatform.buildRustPackage {
+  inherit version src;
+
+  pname = "komac";
+
+  cargoHash = "sha256-J4QZzbyDr4SDt6LlAy9ZdpqgIufZCZHmOC9eu70wMsM=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk_11_0.frameworks.SystemConfiguration
+  ];
+
+  passthru.tests.version = testers.testVersion {
+    package = komac;
+    command = "komac --version";
+    version = src.rev;
```

```suggestion
    inherit version;
```

https://github.com/NixOS/nixpkgs/pull/296350/checks?check_run_id=23406056946

---

🙇‍♂️ 

`src.rev` was also wrong at changelog URL https://github.com/NixOS/nixpkgs/commit/47f04f4679283ff9a39e49cef1486831134acc18#diff-b9450686bfd6f210e22d3ff3963980f47ed5c01b12c9b74630f55b6698f6bab5R39  

Fixed in https://github.com/NixOS/nixpkgs/commit/fe32ad126745b70553aa86e1e52fea02e1bf45b7